R/class-spec.R

```diff
@@ -159,7 +187,9 @@ class_deparse <- function(x) {
 
 class_inherits <- function(x, what) {
   switch(class_type(what),
-    NULL = TRUE,
+    "NULL" = is.null(x),
+    missing = FALSE,
```

Comment:
`class_inherits(missing_class, missing_class) == FALSE`?

Reply:
Yeah, but I don't _think_ that matters